epub33/core/index.html

```diff
@@ -4882,9 +4882,10 @@ <h5>Other <code>nav</code> Elements</h5>
 
 						<p>EPUB Navigation Documents MAY contain one or more <code>nav</code> elements in addition to
 							the <code>toc</code>, <code>page-list</code>, and <code>landmarks</code>
-							<code>nav</code> elements defined in the preceding sections. These additional
-								<code>nav</code> elements SHOULD have an <code>epub:type</code> attribute to provide a
-							machine-readable semantic and MUST have a human-readable heading as their first child.</p>
+							<code>nav</code> elements defined in the preceding sections. If these <code>nav</code>
+							elements are intended for Reading System processing, they MUST have an
+								<code>epub:type</code> attribute with a machine-readable semantic and are subject to the
```

Comment:
`a machine-readable semantic` word missing here?

Reply:
Hm, it's actually kind of redundant to say, since the attribute by definition has machine-readable semantics and is required to have at least one value. I think I'll strip that out entirely.

Reply:
Sounds good! Just thought it read oddly.